metering/metering-uninstall.adoc

```diff
@@ -1,13 +1,27 @@
 :context: metering-uninstall
 [id="metering-uninstall"]
-= Uninstalling Metering
+= Uninstalling metering
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
 You can remove metering from your {product-title} cluster.
 
-include::modules/metering-uninstall.adoc[leveloffset=+1]
+[NOTE]
+====
+Metering does not manage or delete Amazon S3 bucket data. You must manually clean up S3 buckets used to store metering data.
```

Comment:
We have an open BZ for this - should we attach that to this message note, or is it sufficient to make the note without that bug attachment?

Reply:
Going to run this by whoever peer reviews the changes to see how to handle an open BZ that pertains to docs content.